Misc/NEWS.d/next/Documentation/2025-01-14-11-06-41.gh-issue-67206.LYKmi5.rst

```diff
@@ -0,0 +1,3 @@
+Document that :const:`string.printable` is neither printable in the sense of
```

Comment:
I don't think that it's worth it to document documentation changes like this one.

Reply:
As a user, the fact that `string.printable.isprintable()` returns False is surprising. I think more people would read the full changelog rather than the `string.printable` documentation itself (since the name can be self-explanatory even if it's not the case in fact!), so I thought it would at least be a way to make them notice this.

But if think it's not worth, I can leave out the NEWS entry.